.travis.yml

```diff
@@ -5,9 +5,7 @@ os:
   - linux
 
 julia:
-  - 0.7
-  - 1.0
-  - 1.1
+  - 1.2
```

Comment:
I don't see the point in dropping support for earlier Julia versions

Reply:
It is using `test/Project.toml`/`docs/Project.toml` which requires `julia = "1.2"` for the CI. Before that Pkg.jl only supports `extras`/`targets`. I could go back to the `extras` / `targets` if we want to keep testing those versions.

Reply:
Yes I think we should keep testing those versions

Reply:
Should those be `0.7, 1.0, 1.1, 1.2, 1.3, nightly`?

> "Given that v1.1 is no more supported and superseded by v1.2, would it make sense to stop testing packages for v1.1? I'm not totally comfortable spawning dozens of CI jobs for each commit and pull request." - Mosè Giordano 
> "I would say it makes sense." - Kristoffer Carlsson
from Slack this morning, would it make sense to drop `0.7` and `1.1` then?